BismarkUnchained/scenes/Player/Player.gd

```diff
@@ -0,0 +1,101 @@
+extends KinematicBody2D
+
+var accel = 3750
+var max_velocity = 750
+var deadzone = 0.1
+
+var curr_velocity = Vector2(0, 0)
+var controller = false
```

Comment:
`export var` so we can set it in the editor? up to you

Reply:
Yes probably, and maybe part of main.gd